generators/client/templates/webapp/app/components/JhipNavBar.vue.ejs

```diff
@@ -0,0 +1,259 @@
+<template>
```

Comment:
Looks like a small typo in the filename. Guess it should be `JhiNavBar` instead of `JhipNavBar`?